app/controllers/verkiezingen/verkiezingsuitslag/prepare.js

```diff
@@ -1,7 +1,51 @@
 import Controller from '@ember/controller';
 import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { service } from '@ember/service';
+import { getBestuursorganenMetaTtl } from 'frontend-lmb/utils/form-context/bestuursorgaan-meta-ttl';
+import { buildNewMandatarisSourceTtl } from 'frontend-lmb/utils/build-new-mandataris-source-ttl';
+import { syncNewMandatarisMembership } from 'frontend-lmb/utils/sync-new-mandataris-membership';
+
+const CREATE_MODE = 'create';
 
 export default class PrepareInstallatievergaderingController extends Controller {
+  @service store;
+  @service router;
+
+  @tracked editMode = null;
+
+  @action
+  createMandataris() {
+    this.editMode = CREATE_MODE;
+  }
+
+  get isCreating() {
+    return this.editMode === CREATE_MODE;
+  }
+
+  @action
+  cancel() {
+    this.editMode = null;
+  }
+
+  @action
+  async onCreate({ instanceTtl, instanceId }) {
+    this.editMode = null;
+    await syncNewMandatarisMembership(this.store, instanceTtl, instanceId);
+    // TODO not sure why the timeout is needed
+    setTimeout(() => this.router.refresh(), 1000);
```

Comment:
Not sure why the timeout is needed

Reply:
It's because of the resources cache maybe not being updated when the form-content-service returns. This extra second makes it much more likely that this happened. (this is because that is done using deltas and the content service doesn't know whether they have been handled so it can't wait for it)